pycron/__init__.py

```diff
@@ -81,7 +75,7 @@ def start() -> None:
 
             # determine the next time the function should run as specified by its cron string
             next_run_timestamp: int = croniter(scheduled_function.cron_str, last_run).get_next(float)
-            
+
             # check if the function should run based on whether or not the current timestamp
             # meets or exceeds the next run timestamp
             current_timestamp: float = datetime.now().timestamp()
```

Comment:
should not here be `current_timestamp` also be UTC ?

on line 88(82) ur are comparing `current_timestamp` (which is not UTC) with  `scheduled_function.last_run` (with is UTC) + cron delta. looks like bug, no?)